requirements.txt

```diff
@@ -1 +1,4 @@
-cirq-core~=1.1.0
+cirq-core>=1.1.0
+quimb
```

Comment:
This really shouldn't be a core requirement - one can use BGLS without using quimb. The best way to handle this is just to install `quimb` in the notebook. The `cirq_mps_bitstring_probability` function should also be defined in the notebook.

Reply:
Got it I can just reference the bitstring function within the respective notebook then, especially since the matrix product state representation itself is only in the contrib section

Reply:
Awesome I fixed this - it's just in the notebook now and the corresponding test case only runs if it succeeds at importing quimb, in which case it locally defines `cirq_mps_bitstring_probability`.